locator/tablets.hh

```diff
@@ -186,6 +216,9 @@ enum class tablet_transition_kind {
     // The new replica is (tablet_transition_info::next - tablet_info::replicas).
     // The leaving replica is (tablet_info::replicas - tablet_transition_info::next).
     rebuild,
+
+    // Repair the tablet replicas
```

Comment:
isn't that a single `replica`, rather than `replicas`, in plural?

Reply:
A tablet has multiple replicas. We are repairing replicas not a single replica.

Reply:
duh, right.